.gitignore

```diff
@@ -3,6 +3,7 @@ models/
 !src/instructlab/train/lora_mlx/models
 # We recommend using taxonomy here, but not a submodule. Just a work dir.
 taxonomy/
+!src/instructlab/taxonomy/
```

Comment:
what is the point in adding this to the `.gitignore`?

Reply:
the new dir is called `taxonomy` for the taxonomy group

Reply:
Got that, what are we expecting to go into that directory that developers would want to ignore?